base_layer/core_storage_postgres/migrations/2020-03-09-145917_create_merkle_checkpoints/up.sql

```diff
@@ -0,0 +1,14 @@
+create table merkle_checkpoints (
```

Comment:
I think we can do away with checkpoints entirely in an RDMS.

Checkpoints are needed in LMDB as a way to reconstruct history during a re-org. Otherwise they're not needed. 

However, if kernels and outputs store the block they were created in in their tables, a simple query can get us all this information.
I haven't thought this all the way through, but calculating the MMR roots probably also becomes way simpler once we go down this road.